packages/sdk/src/modules/middlewareModule/utils/getRequestSender.ts

```diff
@@ -22,7 +22,7 @@ export const getRequestSender = (options: Options): RequestSender => {
     ssrApiUrl,
     defaultRequestConfig = {},
     methodsRequestConfig = {},
-    cdnCacheBustingId,
+    cdnCacheBustingId = null,
```

Comment:
According to the types - shouldn't it be `undefined`?

```suggestion
    cdnCacheBustingId = undefined,
```